app/src/main/res/layout-sw600dp-port/story_chapter_view.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <import type="org.oppia.android.app.model.ChapterPlayState" />
+
+    <variable
+      name="htmlContent"
+      type="CharSequence" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.android.app.story.storyitemviewmodel.StoryChapterSummaryViewModel" />
+  </data>
+
+  <FrameLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingStart="100dp"
+    android:paddingEnd="100dp">
+
+  <com.google.android.material.card.MaterialCardView
+      android:id="@+id/story_chapter_card"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="28dp"
+      android:layout_marginTop="16dp"
+      android:layout_marginEnd="28dp"
+      android:layout_marginBottom="16dp"
+      android:clickable="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES}"
+      android:onClick="@{(v) -> viewModel.onExplorationClicked()}"
+      app:cardCornerRadius="4dp"
+      app:cardElevation="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? @dimen/elevation_4dp : @dimen/elevation_0dp}">
+
+      <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:clipToPadding="true"
+        android:contentDescription="@{viewModel.name}">
+
+        <org.oppia.android.app.customview.VerticalDashedLine
+          android:layout_width="5dp"
+          android:id="@+id/line"
+          android:layout_height="200dp"
```

Comment:
If I am setting the layout_height here to match_parent then it's not working as expected.